src/python/WMCore/MicroService/Unified/MSManager.py

```diff
@@ -119,6 +121,18 @@ def __init__(self, config=None, logger=None):
                                                           self.logger))
             self.logger.info("=== Running %s thread %s", thname, self.outputProducerThread.running())
 
+        # initialize rule cleaner module
+        if 'ruleCleaner' in self.services:
+            reqStatus = ['announced', 'aborted', 'rejected']
```

Comment:
It should be looking at `aborted-completed` instead of `aborted`

Reply:
done